kvcache/coverage.html

```diff
@@ -0,0 +1,172 @@
+
```

Comment:
I would add this to gitignore

Reply:
Got it. Added to next branch for next PR.

Reply:
I mean I'll fix for this branch